account_invoice_partner_no_autofollow/README.rst

```diff
@@ -0,0 +1,3 @@
+
+By default Odoo adds partner as follower to all invoices.
+This module should disable this behaviour.
```

Comment:
Leave this file blank. Add module description to /readme instead.
Check here: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
Okey)